### PR TITLE
fix ios bug with screensize

### LINKED ISSUE #23: iOS bug with screensize and interactions
right now for ios devices, if you run the tests it doesnt use the right screen size, and so it fails to click or interact with the UI.

### PR DESCRIPTION
**Pull request by alvarolaserna**
closes #23

### CODE REVIEW

**Review by EinarsNG — COMMENTED** [review]
Couple of microscopic typing changes. Apart from that all good.

File: cv_pom/frameworks/testui.py [hunk 1]
```diff
@@ -87,38 +99,51 @@ def _swipe_coordinates(self, coords: tuple = None, direction: str = None):
...
         driver = self._driver.get_driver  # Deprecated property 1.2.1 python-testui
         if inspect.ismethod(self._driver.get_driver):
             driver = self._driver.get_driver()
-        self._driver.actions().w3c_actions = ActionBuilder(
+        if self._driver.device_udid is not None:
+            driver.update_settings({"appium:settings[animationCoolOffTimeout]": duration})
+        actions = ActionBuilder(
             driver,
-            mouse=PointerInput(interaction.POINTER_TOUCH, "touch")
+            mouse=PointerInput(interaction.POINTER_TOUCH, "touch"),
```
Comment (raulbmejias) [inline cv_pom/frameworks/testui.py:140]: This part of the code, looks like, doesn't have the right formatter . There are more part of the code with the same problem.

**Review by raulbmejias — CHANGES_REQUESTED** [review]

File: cv_pom/frameworks/testui.py (hunk 1 above)
Comment (alvarolaserna) [reply cv_pom/frameworks/testui.py:140]: why do you say so? I dont see any difference

Comment (raulbmejias) [reply cv_pom/frameworks/testui.py:140]: The right format should be: (space between "=")
`
mouse = PointerInput(interaction.POINTER_TOUCH, "touch"),`
shouldn't be?
There are other part of the code with that.

Comment (raulbmejias) [reply cv_pom/frameworks/testui.py:140]: Yap, it's a bit confusing but you are right.

**Review by raulbmejias — APPROVED** [review]

**Pull request merged**